testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -167,19 +188,35 @@ private Process startContainer() throws IOException {
         builder.environment().put("KEYCLOAK_ADMIN", "admin");
         builder.environment().put("KEYCLOAK_ADMIN_PASSWORD", "admin");
 
-        if (restart.compareAndSet(false, true)) {
```

Comment:
There is a reason behind the `restart` so that we can track whether subsequent starts can be handled as restarts and not as the first starts.

Or is there a specific reason other than renaming the name of the field?

Reply:
I had 2 AtomicBooleans first, "firstStart" and "restart". You asked me to merge them into one, i did that, but then "restart" doesn't really apply anymore bc. we are also using now this one var for evaluating the first start, see https://github.com/keycloak/keycloak/blob/1eb2fa14b26a976a3959559403ae3fabf4cf700f/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java#L214-L219

Are you now asking me to change back to use 2 vars instead of one? *insert confused face here

Reply:
You don't need to change this, see https://github.com/keycloak/keycloak/pull/13653/files#diff-3006a97a99460ef26e17f2d9e6f2f7807e040b0ecca84c47914f727fc114231eR11.